### DISCUSSION TITLE
Solidus install error

### DISCUSSION THREAD

**Discussion by codingwitharul** [solidusio/solidus#4998] [Troubleshooting]
bin/rails g solidus:install  
**Output**
---
Use `bundle info [gemname]` to see where a bundled gem is installed.
Post-install message from handlebars_assets:
Remember to rake assets:clean or rake assets:purge on update! this is required because of handlebars updates
Post-install message from kt-paperclip:
##################################################
#  NOTE FOR UPGRADING FROM 4.3.0 OR EARLIER      #
##################################################

Paperclip is now compatible with aws-sdk-s3.

If you are using S3 storage, aws-sdk-s3 requires you to make a few small
changes:

* You must set the `s3_region`
* If you are explicitly setting permissions anywhere, such as in an initializer,
  note that the format of the permissions changed from using an underscore to
  using a hyphen. For example, `:public_read` needs to be changed to
  `public-read`.

For a walkthrough of upgrading from 4 to *5* (not 6) and aws-sdk >= 2.0 you can watch
http://rubythursday.com/episodes/ruby-snack-27-upgrade-paperclip-and-aws-sdk-in-prep-for-rails-5
Traceback (most recent call last):
	4: from -e:1:in `<main>'
	3: from /snap/ruby/284/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	2: from /snap/ruby/284/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
	1: from /home/arul/.gem/gems/spring-4.1.1/lib/spring/application/boot.rb:7:in `<top (required)>'
/home/arul/.gem/gems/spring-4.1.1/lib/spring/application/boot.rb:7:in `for_fd': Bad file descriptor - not a socket file descriptor (Errno::EBADF)

---

And stuck at there

**Comment by kennyadsl** [comment]
It's probably related to https://github.com/puma/puma/issues/1499, can you please try the solution suggested on that issue and let us know?
